Gears/script.cpp

```diff
@@ -1139,60 +1141,70 @@ void functionAShift() {
 
     float currSpeed = g_vehData.mWheelAverageDrivenTyreSpeed;
     float currSpeedWorld = g_vehData.mVelocity.y;
-    float nextGearMinSpeed = 0.0f; // don't care about top gear
-    if (currGear < g_vehData.mGearTop) {
-        nextGearMinSpeed = g_settings().AutoParams.NextGearMinRPM * g_vehData.mDriveMaxFlatVel / g_vehData.mGearRatios[currGear + 1];
-    }
+    bool skidding = isSkidding(3.5f);
 
-    float currGearMinSpeed = g_settings().AutoParams.CurrGearMinRPM * g_vehData.mDriveMaxFlatVel / g_vehData.mGearRatios[currGear];
+    if (g_settings().AutoParams.UsingATCU) {
+        AtcuLogic::Cycle();
+    }
+    else {
+        float nextGearMinSpeed = 0.0f; // don't care about top gear
```

Comment:
Additional indenting for the `else` case is a bit annoying. You don't have to do anything about this now, but you might just want to do your stuff and then early-return after handling your stuff

Reply:
not sure about this, you mean
`} else {` ?

Reply:
those formatting styles are given by my VS2019 default... did VS having feature similar like lint? We may use it so the code styles can be unified.